optuna/trial/_frozen.py

```diff
@@ -28,8 +28,9 @@
 class FrozenTrial(BaseTrial):
     """Status and results of a :class:`~optuna.trial.Trial`.
 
-    This object has the same methods as :class:`~optuna.trial.Trial`, and it suggests best
-    parameter values among performed trials. In contrast to :class:`~optuna.trial.Trial`,
+    This object has the same methods as :class:`~optuna.trial.Trial`, and it suggests the same
+    parameter values as in :attr:`params`; it does not suggest fresh parameter values.
```

Comment:
I'm not sure `fresh parameter values` is well defined for users. How about:

```suggestion
    parameter values as in :attr:`params`; it does not sample any value from distribution.
```